lib/wallet.js

```diff
@@ -1,27 +1,17 @@
-import * as ethers from "ethers";
-const bip39 = require("bip39");
-import { getWeb3 } from "./utils.js";
+import { config } from './config.js';
+import * as ethers from 'ethers';
+const bip39 = require('bip39');
+import  { getWeb3 } from './utils.js';
 
 import { Organization } from "./organization.js";
 
-// TODO: move all the provider connection stuff to settings.js
-const ethProvider = process.env.ETH_PROVIDER || "local"; // 'infura', 'etherscan', otherwise will default to local RPC (testrpc, Geth, or Parity..)
```

Comment:
Should these settings be coming from config.js?

Reply:
what do you mean? they are in this PR

Reply:
Sorry, I misread the '-' versus '+'.